charts/sequencer/values.yaml

```diff
@@ -80,9 +80,17 @@ images:
 ports:
   cometBFTP2P: 26656
   cometBFTRPC: 26657
+  cometBFTMETRICS: 26660
   sequencerABCI: 26658
   relayerRPC: 2450
 
+# ServiceMonitor configuration
+serviceMonitor:
+  enabled: true
```

Comment:
Will having this enabled by default cause issues when coreos crds not installed?

Should we add some just commands to install prometheus, and probably by default disable?

Reply:
left in accidentally when testing local, set to false